pkg/fanal/analyzer/language/golang/mod/mod.go

```diff
@@ -123,7 +124,12 @@ func (a *gomodAnalyzer) Version() int {
 }
 
 // fillAdditionalData collects licenses and dependency relationships, then update applications.
-func (a *gomodAnalyzer) fillAdditionalData(apps []types.Application) error {
+func (a *gomodAnalyzer) fillAdditionalData(apps []types.Application, fsys fs.FS) error {
+	absPath, err := filepath.Abs(fsys.(*mapfs.FS).GetUnderlyingRoot())
```

Comment:
If repository contains required files - we try to add them into `CompositeFS`.
e.g. `node_modules` dir for `npm` and `yarn`.
So i suggest to add `vendor` dir into `CompositeFS` (use `Required` function)